src/audio/CMakeLists.txt

```diff
@@ -174,7 +174,7 @@ set(eq-fir_sources module_adapter/module_adapter.c module_adapter/module/generic
 set(eq-iir_sources module_adapter/module_adapter.c module_adapter/module/generic.c eq_iir/eq_iir.c)
 set(dcblock_sources dcblock/dcblock.c dcblock/dcblock_generic.c dcblock/dcblock_hifi3.c dcblock/dcblock_hifi4.c)
 set(crossover_sources crossover/crossover.c crossover/crossover_generic.c)
-set(tdfb_sources tdfb/tdfb.c tdfb/tdfb_generic.c tdfb/tdfb_direction.c)
+set(tdfb_sources module_adapter/module_adapter.c module_adapter/module/generic.c tdfb/tdfb.c tdfb/tdfb_generic.c tdfb/tdfb_direction.c)
```

Comment:
pls dont build the module adapter here, fixed in another PR.

Reply:
There will be CI error from testbench step if I remove this. This can be fixed after merge of #7331.